specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2021-02-10/service.json

```diff
@@ -8320,25 +9267,21 @@
           "description": "The fabric specific object Id of the virtual machine.",
           "type": "string"
         },
-        "initialPrimaryFabricLocation": {
-          "description": "The initial primary fabric location.",
-          "type": "string",
-          "readOnly": true
-        },
-        "initialRecoveryFabricLocation": {
-          "description": "The initial recovery fabric location.",
-          "type": "string",
-          "readOnly": true
-        },
         "initialPrimaryZone": {
           "description": "The initial primary availability zone.",
-          "type": "string",
-          "readOnly": true
+          "type": "string"
+        },
+        "initialPrimaryFabricLocation": {
```

Comment:
mark them as read only

Reply:
done

---
In reply to: [592113436](https://github.com/AsrOneSdk/azure-rest-api-specs/pull/16#discussion_r592113436) [](ancestors = 592113436)